library/src-bootstrapped/scala/quoted/package.scala

```diff
@@ -140,4 +140,9 @@ package object quoted {
     }
   }
 
+  def (tup: T) toExprOfTuple[T <: Tuple] given (ctx: QuoteContext, t: Type[Tuple.ReverseMap[T, Expr]]): Expr[Tuple.ReverseMap[T, Expr]] = {
```

Comment:
This resides outside the given instance with all the other conversions because of #7087.